src/azure-cli-testsdk/azure/cli/testsdk/checkers.py

```diff
@@ -39,6 +39,24 @@ def __call__(self, execution_result):
                                               execution_result.output)
 
 
+class JMESPathCheckGreaterThan(object):  # pylint: disable=too-few-public-methods
```

Comment:
How? e.g. on the http://jmespath.org/ front page example I can do `length(locations)`, how do I express something like `?length(locations) > 0` as a boolean?